src/System.Device.Gpio/System/Device/Gpio/GpioChipException.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Device.Gpio
+{
+    public class GpioChipException : Exception
```

Comment:
This exception is not really holding any additional information, so I would just remove it and be consistent with the exception types that we throw from the other drivers.